### ISSUE TITLE
run.sh shuts down right away

### ISSUE THREAD

**Issue by Calitop** [Creepsky/creepMiner#430]
I have downloaded the latest miner version from the releases page and (1.8.3) and attempted to run it on my Ubuntu 18.04 LTS.

I unpacked the at file and ran the ./run.sh, just like the wiki says. However, instead o starting the web server, the script exists with an error imminently: 
"Pool/Wallet/MiningInfo are all empty! Miner has nothing to do and shutting down!"

This is strange, since I need the webserver to run in order to input in all that info it wants (like pool, wallet etc) through the web interface, as per wiki instructions.

So basically, the script requires the config file to be filled in, before it runs the web server, but I need to run the webserver first, in order to fill in that info, so it can generate the config file.

So, Is there any way to get past this bug and force the web server to run with the empty config file?

**Comment by zodi1337** [comment]
Mh strange..
But there should be no need of the webserver to generate your mining.conf.
Which version you tried? creepMiner-1.8.3.0-Linux.tar.gz or  creepMiner-opencl-1.8.3.0-Linux.tar.gz?
Did you ./run.sh in terminal or from desktop?

Normaly after downloading and unpacking there is no mining.conf
Here when i do a ./run.sh in terminal, creepminer ask me in the console for the setup and starts to build the mining.conf.
Is there already a mining.conf in your creepminer directory? If yes, try to delete and ./run.sh again from console.

**Comment by Calitop** [comment]
I tried both the opencl and the normal one, both with the same result.

I also deleted the .creepMiner directory it made in my home directory, from which it load the config, but after I run the run.sh (i run it from terminal) It just creates that directory again and I get the exact same result.

**Comment by RokyErickson** [comment]
I’m having the exact same issue. I’m on archlinux I’m using the Opencl version. I compiled the source code and installed which generated a ~/creepMiner/bin/creepMiner which I ran and i get the same error he does? Is this a bug or am I initializing wrong?

**Comment by fiddybux** [comment]
Do the precompiled versions exhibit the same behaviour?
